### PR TITLE
add some unicodetools snapshots after svn r1566

### PR DESCRIPTION
**Pull request by markusicu**
- started from unicodetools master which had history up to unicodetools svn r1566
- replaced all files with Laurentiu's svn workspace snapshot from svn r1804
- same for five more snapshots from Peter/Laurentiu/Markus/Mark, up to svn r1830 (mostly somewhat mixed svn revision numbers)
- used unzip -a to convert CR LF line endings to just LF for assembly on Linux
- converted CR and CR LF line endings to LF in more files
- removed executable bit from .svg and .txt files
- omitted UnicodeJsps/build/ which mostly had Java .class files
- omitted UnicodeJsps/UnicodeJsps.war
- omitted ".DS_Store" macOS metadata files

### CODE REVIEW

File: unicodetools/data/uca/4.1.0/CollationTest/CollationTest_NON_IGNORABLE.txt [hunk 1]
```diff
@@ -1437,7 +1437,7 @@ FE13 0062;	# (︓) PRESENTATION FORM FOR VERTICAL COLON	[023E 0F85 | 0020 0020 |
 0017 0021;	# ('') <END OF TRANSMISSION BLOCK>	[0253 | 0020 | 0002 |]
 0018 0021;	# ('') <CANCEL>	[0253 | 0020 | 0002 |]
 0019 0021;	# ('') <END OF MEDIUM>	[0253 | 0020 | 0002 |]
-001A 0021;	# ('') <SUBSTITUTE>	[0253 | 0020 | 0002 |]
+001A 0021;	# ('') <SUBSTITUTE>	[0253 | 0020 | 0002 |]
```
Comment (pedberg-icu) [inline unicodetools/data/uca/4.1.0/CollationTest/CollationTest_NON_IGNORABLE.txt]: FYI the change in this line and others like it below is to change from having no actual character between the single quotes ('') to having that actual 0x1A SUBSTITUTE character (invisible) between the quotes (''). Seems like it was likely an intentional change in the source (not an artifact of reconstruction), but is in a comment anyway.

Comment (markusicu) [reply unicodetools/data/uca/4.1.0/CollationTest/CollationTest_NON_IGNORABLE.txt]: I know. U+001A is "end of file" on DOS/Windows, and I guess these characters disappear when you are on a Windows machine. I had started with files from Laurentiu, and there were so many changes that stuff like this has likely snuck in. I will watch out for this in the next snapshot.

See issue #2.

**Comment by pedberg-icu** [comment]
The overall diffs cannot be generated by github (3241 files chnaged) but it is possible to look at the diffs for the individual commits. I looked at the ones relevant for the change from r1804 to r1805 (my version being r1805) and they seem fine.

**Comment by markusicu** [comment]
going once, going twice, ... ok for me to merge?

**Comment by pedberg-icu** [comment]
> going once, going twice, ... ok for me to merge?

Yes, thanks for all of the work. This seems like the best possible reconstruction we can do.

**Pull request merged**